### ISSUE TITLE
tagSubscriber not working

### ISSUE THREAD

**Issue by pieterjandesmedt** [DripEmail/drip-nodejs#13]
Everything you put in the payload results in these errors:
```javascript
[ { code: 'email_error',
              attribute: 'email',
              message: 'Email must be a valid email address' },
            { code: 'presence_error',
              attribute: 'email',
              message: 'Email can\'t be blank' },
            { code: 'length_error',
              attribute: 'tag',
              message: 'Tag is too short (minimum is 1 character)' } ]
```
Turns out the payload shoudn't be put in an array.
Fixed by #12

**Comment by samudary** [comment]
Update published - v2.0.1

**Comment by pieterjandesmedt** [comment]
Thanks!
